### PR TITLE
Invite resources.

### PR DESCRIPTION
**Pull request by CoreyShupe**
Addresses https://github.com/princesslana/ErisCasper.java/issues/127

### CODE REVIEW

File: src/main/java/com/github/princesslana/eriscasper/rest/InviteRoute.java [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    this.path = "/invites/" + code;
+  }
+
+  /**
+   * @see <a href="https://discordapp.com/developers/docs/resources/invite#get-invite">
+   *     https://discordapp.com/developers/docs/resources/invite#get-invite</a>
+   */
+  public Route<Void, Invite> get(Optional<Boolean> withCounts) {
+    return Route.get(
+        path + withCounts.map(bool -> ";with_counts=" + bool).orElse(""), Invite.class);
```
Comment (ianagbip1oti) [inline src/main/java/com/github/princesslana/eriscasper/rest/InviteRoute.java]: Is this mean to be a ? instead of ;

Comment (ianagbip1oti) [reply src/main/java/com/github/princesslana/eriscasper/rest/InviteRoute.java]: Why not use QueryStringBuilder?

Comment (CoreyShupe) [reply src/main/java/com/github/princesslana/eriscasper/rest/InviteRoute.java]: Nope
https://stackoverflow.com/questions/39266970/what-is-the-difference-between-url-parameters-and-query-strings
https://discordapp.com/developers/docs/resources/invite#get-invite

Comment (CoreyShupe) [reply src/main/java/com/github/princesslana/eriscasper/rest/InviteRoute.java]: It's not a QueryString

**Review by ianagbip1oti — APPROVED** [review]

**Pull request merged**